src/main/java/oshi/software/os/mac/MacOperatingSystem.java

```diff
@@ -26,4 +27,14 @@ public MacOperatingSystem() {
         this.family = System.getProperty("os.name");
         this.version = new MacOSVersionInfoEx();
     }
+
+    @Override
+    public long getOpenDescriptors() {
+        return SysctlUtil.sysctl("kern.num_files", -1);
```

Comment:
Disregard above comment. The type that sysctl returns is int, so this is correct